src/content/enterprise/administration/volume-snapshots.mdx

```diff
@@ -312,4 +312,4 @@ spec:
 
 As part of the creation of the persistent volume claim, Okteto will import the specified snapshot into your cluster using a `VolumeSnapshotContent` and will set the source of your persistent volume claim to this [VolumeSnapshotContent](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), ready to be used by your development environment.
 
-Allowing volume snapshots from a snapshot-id is enabled by default, you can disable this feature by changing the configuration `allowIDAnnotation` to false in case you want to prevent PVCs created from any snapshot-id at the cluster.
\ No newline at end of file
+By default, anyone can use any of your cloud provider's snapshot ID as a data source. You can disable this by changing the configuration `allowIDAnnotation ` to `false` in the `volumeSnapshots` section of the configuration.
```

Comment:
@teresaromero when this feature is disabled, what option do the user has then? only use the dev.okteto.com/from-snapshot-namespace annotation?